schemas/options.yaml

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: GPL-2.0-only OR BSD-2-clause
+# Copyright 2021 Google LLC
+#
+# Based on the precedent set by IEEE 1275-1994 IEEE Standard for Boot
+# (Initialization Configuration) Firmware: Core Requirements and Practices
+# https://www.openfirmware.info/data/docs/of1275.pdf
+
```

Comment:
This is worth being in the description, not a comment. Someday we might generate documentation from this.

Reply:
done